installer.tpl.sh

```diff
@@ -445,7 +445,7 @@ main ()
     install_yum
   elif [ ${os} = "redos" ] && [[ ${dist} = "7.3" ]]; then
     echo "Setting up yum repository... "
-    install_yum
+    install_dnf
```

Comment:
This function enables `tarantool_modules` repo, which doesn't exist for RED OS. Please check the `install_yum` function.

Reply:
Default parameter `tarantool_modules` is disabled and in `install_yum` script enables this for some distros but not for RedOS.

Reply:
You should fix `install_dnf` to avoid this https://github.com/tarantool/installer.sh/actions/runs/6801362627/job/18492043779?pr=35#step:7:29. One more time, check `install_yum` function and see how it is done there.

Reply:
fixed